src/components/AppList/index.scss

```diff
@@ -1,25 +1,26 @@
 @import '~scss/vars';
 
-.appList {
-  a {
+
+.appList, .categoryApps {
+  overflow: hidden;
+  _zoom: 1;
```

Comment:
`_zoom` ? typo

Reply:
Clear float style ,  _zoom for compatible IE,  detail see https://www.cnblogs.com/java-zone/articles/2733991.html.